environment.yml

```diff
@@ -0,0 +1,35 @@
+# Usage: `conda env update -f environment.yml`
+channels:
+    - conda-forge
+    - pytorch
+
+dependencies:
+    - python==3.7.6
```

Comment:
does `conda` supports reading from a file, to have here `-r requirements.txt`?
personally I do not like having the dependency packages in two different formats, then maintenance is harder (do not forget to do updates in two separate places)

Reply:
I know but I don't think there are other alternatives here. I will open a new PR for conda env file. I think it's good to have a conda env file (this is what I use to set up my dev PL env). But I agree it's not ideal having deps in separate files. That being said, it's only two files.

Reply:
sure and unless we need to skip any version if shall be quite stable since we have only lowest version... :]